### PR TITLE
[WFCORE-5246] Bump management API version to 16.0.0

### PR DESCRIPTION
**Pull request by jmesnil**
and bump the XSD namespace to `urn:jboss:domain:16.0`

JIRA: https://issues.redhat.com/browse/WFCORE-5246

Signed-off-by: Jeff Mesnil <jmesnil@redhat.com>

### CODE REVIEW

File: server/src/main/java/org/jboss/as/server/parsing/StandaloneXml_16.java [hunk 1]
```diff
@@ -0,0 +1,1031 @@
...
+import org.jboss.as.server.mgmt.NativeManagementResourceDefinition;
+import org.jboss.as.server.services.net.SocketBindingGroupResourceDefinition;
+import org.jboss.dmr.ModelNode;
+import org.jboss.dmr.ModelType;
+import org.jboss.dmr.Property;
+import org.jboss.staxmapper.XMLExtendedStreamReader;
+import org.jboss.staxmapper.XMLExtendedStreamWriter;
+
+/**
+ * Parser and marshaller for standalone server configuration xml documents (e.g. standalone.xml) that use the urn:jboss:domain:15.0 schema.
```
Comment (yersan) [inline server/src/main/java/org/jboss/as/server/parsing/StandaloneXml_16.java:100]: nitpick s/urn:jboss:domain:15.0/urn:jboss:domain:16.0

File: host-controller/src/main/java/org/jboss/as/host/controller/parsing/DomainXml_16.java [hunk 2]
```diff
@@ -0,0 +1,1093 @@
...
+import org.jboss.as.server.controller.resources.DeploymentAttributes;
+import org.jboss.as.server.parsing.CommonXml;
+import org.jboss.dmr.ModelNode;
+import org.jboss.dmr.ModelType;
+import org.jboss.dmr.Property;
+import org.jboss.staxmapper.XMLExtendedStreamReader;
+import org.jboss.staxmapper.XMLExtendedStreamWriter;
+
+/**
+ * Parser and marshaller for domain-wide configuration xml documents (e.g. domain.xml) that use the urn:jboss:domain:15.0 schema.
```
Comment (yersan) [inline host-controller/src/main/java/org/jboss/as/host/controller/parsing/DomainXml_16.java:104]: nitpick s/urn:jboss:domain:15.0/urn:jboss:domain:16.0

File: host-controller/src/main/java/org/jboss/as/host/controller/parsing/HostXml_16.java [hunk 3]
```diff
@@ -0,0 +1,1801 @@
...
+import org.jboss.as.server.parsing.SocketBindingsXml;
+import org.jboss.as.server.services.net.SocketBindingGroupResourceDefinition;
+import org.jboss.dmr.ModelNode;
+import org.jboss.dmr.ModelType;
+import org.jboss.dmr.Property;
+import org.jboss.staxmapper.XMLExtendedStreamReader;
+import org.jboss.staxmapper.XMLExtendedStreamWriter;
+
+/**
+ * Parser and marshaller for host controller configuration xml documents (e.g. host.xml) that use the urn:jboss:domain:15.0 schema.
```
Comment (yersan) [inline host-controller/src/main/java/org/jboss/as/host/controller/parsing/HostXml_16.java:124]: nitpick s/urn:jboss:domain:15.0/urn:jboss:domain:16.0

**Review by yersan — APPROVED** [review]
It looks good to me, I only found minor documentation details carried on from the previous files.

File: server/src/main/java/org/jboss/as/server/parsing/StandaloneXml_16.java (hunk 1 above)
Comment (jmesnil) [reply server/src/main/java/org/jboss/as/server/parsing/StandaloneXml_16.java]: fixed, thanks!

File: host-controller/src/main/java/org/jboss/as/host/controller/parsing/HostXml_16.java (hunk 3 above)
Comment (jmesnil) [reply host-controller/src/main/java/org/jboss/as/host/controller/parsing/HostXml_16.java]: fixed, thanks!

File: host-controller/src/main/java/org/jboss/as/host/controller/parsing/DomainXml_16.java (hunk 2 above)
Comment (jmesnil) [reply host-controller/src/main/java/org/jboss/as/host/controller/parsing/DomainXml_16.java]: fixed, thanks!

**Pull request merged**